core/src/main/scala/chisel3/VerificationStatement.scala

```diff
@@ -6,6 +6,8 @@ import chisel3.experimental.{BaseModule, SourceInfo}
 import chisel3.internal.Builder.pushCommand
 import chisel3.internal._
 import chisel3.internal.firrtl.ir._
+import chisel3.layer.block
+import chisel3.layers
```

Comment:
It's sort of weird that we have both the `layer` and `layers` packages... Not really this PR but maybe we should just move the default layers to package `layer`? Idk, maybe it's just something to document. I do see utility in having the default layers in a separate package from the core `layer` APIs.

Reply:
Rare @ucbjrl "👍". 🐝 🍯 

Yeah, this can probably be collapsed down to a single package. The motivation was that `layer` is APIs for creating singular things: `object Foo extends layer.Layer` is creating _a_ layer or `layer.block(Foo)` is creating _a_ layer block. The `layers` package actually contains the plural layers. This isn't strong reasoning other than the imports vibed better in my head this way.

`layers` is going into the next major release, so this can change anytime.